src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -135,6 +143,9 @@ private[query] class EventsByTagPublisher(
 
     case Cancel =>
       context.stop(self)
+
+    case tagWritten: String if tagWritten == tag =>
```

Comment:
sure, but there are also other "states"
either override unhandled, or handle/ignore it in all states